### ISSUE TITLE
[bug]: Python runs, but extremely slow

### ISSUE THREAD

**Issue by DomPfaff** [invoke-ai/InvokeAI#3270]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Windows

### GPU

cuda

### VRAM

4GB

### What version did you experience this issue on?

2.3.4.post1

### What happened?

There is a strange error that happens not only with invokeai itself, it happens with the standalone version on invokeai too.
At the beginning of this month, i found invokeai and was verry happy that i worked flawlessly on 4GB Vram. I had zero problems, even on sizes like 1024x1024. It worked really like a charm. Waiting 60 seconds for results to show up was absolutely no problem at all for me. But after only one week, generating images got extremely frustrating. After starting the webgui again, python was running and doing nothing at all. So i tried to reduce the image size step by step. Nothing happened. After i changed the size for the last one 64x64, i waited patiently. After more than 1 hour, the command windows showed some reaction and started to generate an image. Nice, but it took 5 minutes, which isn't normal at all. After that, i waited again and again, after 1 hour i got a new image. After that, my frustration was just at its highest point and i closed the tool and removed it completely, including python. I already downloaded the standalone version of involeai and was happy at that time, that it worked, but only on 768x768 resolution. But after one week, i ran into the exact same problems like with the original invokeai.
Long story short:

python is running but eating up my RAM and VRAM like crazy and the tool is doing nothing. 
And one thing, i forgot to mention: there is no error message and the "free VRAM after generating images" option is active..

Sorry for this very long post, but i wanted to explain everything as easy as i could for you to nderstand what is going on.

**Comment by DomPfaff** [comment]
i forgot to mention, that i reinstalled the original invokeai to test it again, but it is still the same. slow and not really working at all

**Comment by DomPfaff** [comment]
alright.. after i got home from work, i tried to recall a few steps. and the most important step was to re-download the version of invoke-ai that worked. and it was 2.3.2.post1...
it is working again like it is supposed to.. something is still not correct with this new version as it only produces cO2 instead of images...

**Comment by DomPfaff** [comment]
tested image generation with other tools and they work absolutely perfect and give me results after 30 seconds maximum. invokeai, even after updating to beta build, doesn' give me one result even after 2 hours... 

invokeai was a nice experience, but right now, this tool is absolutely useless and goes right into the trash bin. maybe, after you guys have watched other developers and learned what you have to do, this tool will get a next chance....
